src/extension.ts

```diff
@@ -856,16 +856,7 @@ function createIdfTerminal() {
         const espIdfTerminal = vscode.window.createTerminal({ name: "ESP-IDF Terminal", env: modifiedEnv,
                                                               cwd: workspaceRoot.fsPath });
         espIdfTerminal.show();
-        espIdfTerminal.sendText(`${envSetCmd} IDF_PATH=${modifiedEnv.IDF_PATH}`);
-        let IDF_ADD_PATHS_EXTRAS =
-            `${path.join(modifiedEnv.IDF_PATH, "components", "esptool_py", "esptool")}`;
-        IDF_ADD_PATHS_EXTRAS =
-            `${IDF_ADD_PATHS_EXTRAS}${path.delimiter}${path.join(modifiedEnv.IDF_PATH,
-                                                                 "components", "espcoredump")}`;
-        IDF_ADD_PATHS_EXTRAS =
-            `${IDF_ADD_PATHS_EXTRAS}${path.delimiter}${path.join(modifiedEnv.IDF_PATH,
-                                                                 "components", "partition_table")}`;
-        espIdfTerminal.sendText(`${envSetCmd} PATH=${IDF_ADD_PATHS_EXTRAS}${path.delimiter}${modifiedEnv.PATH}`);
+        espIdfTerminal.sendText(`${envSetCmd} IDF_PATH="${modifiedEnv.IDF_PATH}"`);
```

Comment:
Keep in mind that for cmd.exe the quoting syntax is
```
set "PATH=..."
```
unlike export syntax in bash:
```
export PATH="..."
```

Reply:
Oh thanks, sorry about that.